src/main/webapp/app/components/newAccountForm/NewAccountForm.tsx

```diff
@@ -463,6 +464,9 @@ export class NewAccountForm extends React.Component<INewAccountForm> {
                       ACCOUNT_TITLES.POSITION,
                       this.selectedLicense
                     )}
+                    validate={{
+                      ...OPTIONAL_TEXT_VAL,
+                    }}
```

Comment:
The `jobTitle` field in UserPage.tsx should also have this length validation. Currently it is set to SHORT_TEXT_VAL.

Reply:
good catch... I'll fix that